src/olympia/versions/models.py

```diff
@@ -222,7 +222,16 @@ def __str__(self):
         return markupsafe.escape(self.version)
 
     @classmethod
-    def from_upload(cls, upload, addon, selected_apps, channel, parsed_data=None):
+    def from_upload(
+        cls,
+        upload,
+        addon,
+        channel,
+        *,
+        selected_apps=None,
+        compatibility=None,
+        parsed_data=None,
+    ):
         """
         Create a Version instance and corresponding File(s) from a
         FileUpload, an Addon, a list of compatible app ids, a channel id and
```

Comment:
Could you reword that doc to match the parameters ?

Reply:
https://github.com/mozilla/addons-server/pull/18071/commits/b6f27f0a754fe5214e2cdc474c3af2478c4dfd74